addons/mail/static/src/new/core_ui/message.js

```diff
@@ -131,6 +132,17 @@ export class Message extends Component {
         return this.props.message;
     }
 
+    get body() {
+        if (
+            this.message.subtypeDescription &&
+            this.message.subtypeDescription.toLowerCase() !==
+                htmlToTextContentInline(this.message.body).toLowerCase()
+        ) {
+            return markup(this.message.body + this.message.subtypeDescription);
```

Comment:
You should markup `subtypeDescription` as soon as when you receive the data, like we did in `body` for message.
If needed ofc.

Do not concatenate and then wrap with `Markup()`. Instead put 2 `t-out` in template (one could be conditional)